site/content/integrate/apps/_index.md

```diff
@@ -33,53 +33,53 @@ When you develop using the Apps Framework, your apps can:
 
 ### What's the difference between the Apps framework and the plugin framework?
 
-The Apps Framework provides a few differences from plugins, including:
+The Apps framework provides a few differences from plugins, including:
 
-- Interactive elements are easier to use and develop.
-- Compatible with both Mobile and Desktop clients without any extra code.
+-   Interactive elements are easier to use and develop.
+-   Compatible with both Mobile and Desktop clients without any extra code.
 
-### What language should I use to write apps?
+### What language should I use to write Apps?
 
 Any language you want. We currently have an [official driver for Go](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps), and we are planning to release other drivers in the future in conjunction with our community.
 
-### How and where will my app be hosted?
+### How and where will my App be hosted?
 
-1. **[Mattermost Cloud](https://mattermost.com/mattermost-cloud/)** apps. Cloud
-   customers can use apps from Mattermost Marketplace. Marketplace apps are
+1. **[Mattermost Cloud](https://mattermost.com/mattermost-cloud/)** Apps. Cloud
+   customers can use Apps from Mattermost Marketplace. Marketplace Apps are
    deployed as AWS Lambda functions, and their usage is included in the service.
-   You can find more information about the Marketplace [here](https://mattermost.com/marketplace/). Example AWS Lambda apps can be
+   You can find more information about the Marketplace [here](https://mattermost.com/marketplace/). Example AWS Lambda Apps can be
    found
    [here](https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/js/aws_hello)
    (JavaScript) and
    [here](https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-serverless)
    (go).
-2. **External** (HTTP) apps. Apps can be hosted as publicly (or privately) available HTTP
+2. **External** (HTTP) Apps. Apps can be hosted as publicly (or privately) available HTTP
    services, the choice of hosting provider is yours. A self-managed Mattermost
-   customer would be able to install your app from a URL. External apps are not
+   customer would be able to install your App from a URL. External Apps are not
    currently accepted in the Mattermost Marketplace.
-3. **Customer-deployable** apps. An app can be packaged as a bundle, deployable
+3. **Customer-deployable** Apps. An App can be packaged as a bundle, deployable
    by the customer in their own hosting environment. Currently, [AWS
    Lambda](https://aws.amazon.com/lambda/),
    [OpenFaaS](https://www.openfaas.com/), and [Kubeless](https://kubeless.io/)
    are supported, with plans for more serverless platforms, Kubernetes, and
-   docker compose. Customer-deployable apps are not yet distributed via the
+   docker compose. Customer-deployable Apps are not yet distributed via the
```

Comment:
To me, deploying via docker compose is the same as deploying an HTTP server. Each language/environment is different and requires a different setup for deployment outside of the automated lambda function strategy. Not sure if anything needs to change here though.